geth/node/manager_test.go

```diff
@@ -196,6 +197,37 @@ func (s *ManagerTestSuite) TestReferences() {
 	}
 }
 
+func (s *ManagerTestSuite) TestNodeSyncPoll() {
+	require := s.Require()
+	require.NotNil(s.NodeManager)
+
+	nodeConfig, err := MakeTestNodeConfig(params.RopstenNetworkID)
+	require.NoError(err)
+
+	require.False(s.NodeManager.IsNodeRunning())
+	_, err = s.NodeManager.StopNode()
+	require.EqualError(err, node.ErrNoRunningNode.Error())
+
+	require.False(s.NodeManager.IsNodeRunning())
+	nodeStarted, err := s.NodeManager.StartNode(nodeConfig)
+	require.NoError(err)
+
+	<-nodeStarted // wait till node is started
+	require.True(s.NodeManager.IsNodeRunning())
+
+	lightEth, err := s.NodeManager.LightEthereumService()
+	require.NoError(err)
+	require.NotNil(lightEth)
+
+	syncer := node.NewSyncPoll(lightEth)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	defer cancel()
+
+	err = syncer.Poll(ctx)
+	require.NoError(err)
+}
```

Comment:
Are you sure we need this test?
1. If there's an error with `syncer.Poll`, we'll see it in every test.
2. There's no guarantee that this test will be run the first. Thus any other test requiring synced blockchain will run this code anyway.

Reply:
I guess that is indeed true, also since we have https://github.com/status-im/status-go/blob/f5ea1eef38cccadba9fe706b5a96a44b8e0712b2/geth/testing/testing.go#L111, we can remove this.

Once #375 is merged, will refactor and update PR.